### PR TITLE
Fozzie

### PR DESCRIPTION
**Pull request by mkunkel**
Verifies that fozzie is in the Gemfile

### CODE REVIEW

File: lib/bobo.rb [hunk 1]
```diff
@@ -1,11 +1,12 @@
 require 'bobo/version'
 require 'bobo/server'
 require 'bobo/current_version'
+require 'bobo/metrics'
 require 'open3'
 
 module Bobo
   def self.ready?(opts = {})
-    results = [Bobo::Server, Bobo::CurrentVersion].map { |klass| klass.new.execute(opts) }
+    results = [Bobo::CurrentVersion, Bobo::Server, Bobo::Metrics].map { |klass| klass.new.execute(opts) }
```
Comment (edubkendo) [inline lib/bobo.rb]: If you namespace your classes one deeper, so all the classes you are loading here are under a namespace like Bobo::Tests you could do `Bobo::Tests.constants.map` I think. Which will be more obviously cleaner as this array grows.

File: lib/bobo/metrics.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+module Bobo
+  class Metrics < Base
+    def process
+      return [] unless File.exist?('./Gemfile')
```
Comment (edubkendo) [inline lib/bobo/metrics.rb]: This is a fine way to check this, I think, but if you will be checking the Gemfile much, Bundler actually has a programatic api you can use to get the dependencies. I used it in rsense.

**Comment by edubkendo** [comment]
A couple suggestions but overall:

haruhithumbsup

File: lib/bobo.rb (hunk 1 above)
Comment (edubkendo) [inline lib/bobo.rb]: Ah may have to do: `Foo.constants.map(&Foo.method(:const_get)).grep(Class)` to get the actual class object and ignore any other constants defined inside those classes..

Comment (mkunkel) [inline lib/bobo.rb]: Good point. I've made that change, but would like to do that as a separate PR. The change is a bit larger than I would like when combined with this PR.

Other than this comment, does everything look OK?

Comment (mkunkel) [inline lib/bobo.rb]: For posterity... This got a thumb in Slack

File: lib/bobo/metrics.rb (hunk 2 above)
Comment (mkunkel) [inline lib/bobo/metrics.rb]: Interesting. I'll take a look into that, I have at least one more case where I will need to check dependencies.
